### PR TITLE
Color theme preparation

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/3649

Deleted a bunch of unused files and cleaned out older colors/typography and consolidated some files.

### CODE REVIEW

**Review by bthaile — APPROVED** [review]
lgtm, I want all the UI devs to look at this PR

**Review by phoebemirman — APPROVED** [review]

File: packages/augur-ui/src/assets/styles/_animations.less [hunk 1]
```diff
@@ -32,8 +32,8 @@ body {
   .animation(flash-slow @animation-speed-slow @animation-default-type infinite alternate);
 
   .keyframes(flash-slow, {
-    0% { background: fade(@color, @amount-subtle) }
-    100% { background: fade(@color, @amount-light) }
+    0% { background: fade(@color, 2%) }
```
Comment (bconfortin) [inline packages/augur-ui/src/assets/styles/_animations.less:35]: @JohnDanz Why don't you modify `@amount-subtle` and `@amount-light` directly?
If it's being used elsewhere, I suggest creating another variable, specially if the same values (2% and 15%) will be used throughout the code.

Comment (bthaile) [reply packages/augur-ui/src/assets/styles/_animations.less:35]: looks like @amount-subtle isn't needed at all and isn't used anywhere else.

**Review by bconfortin — APPROVED** [review]
I left a suggestion on the code. I am approving this PR because it's not something critical.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌶

 This link will always point to this PR's latest changes: https://peek.run/2n5hlv6

**Pull request merged**